### PR TITLE
Clean up targets

### PR DESCRIPTION
**Pull request by unDemian**
## Summary
Based on this discussion https://github.com/Automattic/happychat-client/pull/178#issuecomment-377899480 we decided to clean up the targets a bit.

## Testing
`npm start` should run the standalone example, `npm run build:standalone` should built production ready js/css assets.

### CODE REVIEW

**Comment by nosolosw** [comment]
Thanks for this! It makes everything more clear. I like how this work is coming along.

File: package.json [hunk 1]
```diff
@@ -14,7 +14,7 @@
     "> 1%"
   ],
   "scripts": {
-    "preanalyze-bundles": "webpack --config webpack.cdn.config.js --profile --json > stats.json",
+    "preanalyze-bundles": "webpack --config webpack.standalone.config.js --profile --json > stats.json",
```
Comment (nosolosw) [inline package.json]: I guess this needs to be updated with the changes at https://github.com/Automattic/happychat-client/pull/191?

File: src/config/development.json [hunk 2]
```diff
@@ -2,6 +2,7 @@
 	"css_url": "http://localhost:9000/happychat.css",
 	"happychat_url": "https://happychat-io-staging.go-vip.co/customer",
 	"twemoji_cdn_url": "https://s0.wp.com/wp-content/mu-plugins/wpcom-smileys/twemoji/2/",
+	"oauth_client_id": "54006",
```
Comment (nosolosw) [inline src/config/development.json]: As per the comments https://github.com/Automattic/happychat-client/pull/188#discussion_r178524068 I guess we are also removing this?

File: package.json (hunk 1 above)
Comment (unDemian) [reply package.json]: It will be updated when merging the other branch.

File: src/config/development.json (hunk 2 above)
Comment (unDemian) [reply src/config/development.json]: Indeed, it will be updated when merging the other branch.

File: webpack.standalone.config.js [hunk 3]
```diff
@@ -36,6 +36,9 @@ switch ( env ) {
 		config.devtool = 'source-map';
 		config.devServer = {
 			contentBase: path.resolve( __dirname, 'targets/standalone' ),
+			historyApiFallback: {
+				index: 'example.html',
```
Comment (nosolosw) [inline webpack.standalone.config.js]: I think the example HTML and assets are missing?

Comment (unDemian) [reply webpack.standalone.config.js]: Forgot to push them. It should be ok now

**Pull request merged**